$NetBSD: patch-aa,v 1.1.1.1 2003/11/19 02:13:41 jeremy-c-reed Exp $

The LANGUAGES patch is because of my broken msgconv. - reed

--- configure.orig	2003-10-07 07:51:53.000000000 -0700
+++ configure
@@ -1706,7 +1706,7 @@ echo "${ECHO_T}no" >&6
 fi
 
 
-LANGUAGES="hu zh_CN cs da_DK de es fr it nl pl pt_BR ro ru sv"
+LANGUAGES="zh_CN cs de nl ro ru"
 
 
 DISTLANGS=
@@ -1756,7 +1756,7 @@ if test "${am_cv_pathless_PYTHON+set}" =
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
 
-	for am_cv_pathless_PYTHON in python python2 python2.2 python2.1 python2.0 python1.6 python1.5 : ; do
+	for am_cv_pathless_PYTHON in python python2 python2.3 python2.2 python2.1 python2.0 python1.6 python1.5 : ; do
           if test "$am_cv_pathless_PYTHON" = : ; then
             { { echo "$as_me:$LINENO: error: no suitable Python interpreter found" >&5
 echo "$as_me: error: no suitable Python interpreter found" >&2;}
